### PR TITLE
[wmi] `namespace` support

### PR DESCRIPTION
**Pull request by yannmh**
Collect WMI metrics from the specified namespace.

### CODE REVIEW

File: tests/checks/mock/test_wmi_check.py [hunk 1]
```diff
@@ -170,9 +182,31 @@ def setUp(self):
                     Mocked_Win32_Service(**Win32_PerfFormattedData_PerfProc_Process_attr),
             })
 
+    @patch('wmi.WMI')
+    def test_wmi_conn(self, mocked_wmi_conn):
+        """
+        Establish a WMI connection to the specificied host/namespace, with the right credentials.
+        """
+        # Reload `wmi` to use the mock
+        import wmi
```
Comment (irabinovitch) [inline tests/checks/mock/test_wmi_check.py]: flake8 thinks this might not be in use.

Comment (yannmh) [inline tests/checks/mock/test_wmi_check.py]: Yep it's useless. Removing it. Thanks @irabinovitch

**Comment by yannmh** [comment]
I fixed the tests. @JohnLZeller would you like to take a pass on it ?

**Comment by yannmh** [comment]
@simnyc this will be part of the 5.5.0 release, as discussed before.

**Comment by JohnLZeller** [comment]
lgtm, nice work on those tests

**Pull request merged**